app/controllers/application_controller.rb

```diff
@@ -1,6 +1,9 @@
 class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
-
+  protect_from_forgery with: :exception
+  def hello
```

Comment:
Why this method?

Reply:
I was trying to render the page... ...

Reply:
Why? There ain’t need for it. Since root is fruits#index. Remove it